src/wallet/gtest/test_wallet_cert.cpp

```diff
@@ -0,0 +1,124 @@
+#include <gtest/gtest.h>
+#include "gtest/tx_creation_utils.h"
+#include <boost/filesystem.hpp>
+
+#include <chainparams.h>
+#include <wallet/wallet.h>
+#include <wallet/walletdb.h>
+
+
+class CertInWalletTest : public ::testing::Test {
+public:
+    CertInWalletTest(): walletDbLocation(), pWallet(nullptr), pWalletDb(nullptr) {}
+    ~CertInWalletTest() = default;
+
+    void SetUp() override {
+        //Setup environment
+        SelectParams(CBaseChainParams::TESTNET);
+        boost::filesystem::path walletDbLocation = boost::filesystem::temp_directory_path() / boost::filesystem::unique_path();
```

Comment:
use a different name, just to disambiguate with data member and improve readability

Reply:
Actually I intended yo use the data member, removing declaration here